randovania/games/super_metroid/logic_database/Crateria.txt

```diff
@@ -127,23 +127,36 @@ Extra - asset_id: 3
   > Door to West Ocean
       Any of the following:
           Grapple Beam or Space Jump
-          # Short charge shinespark
-          Speed Booster and Short Charge (Intermediate) and Disabled Door Lock Randomizer and Disabled Entrance Randomizer
-          # Gravity Jump
-          Gravity Suit and Gravity Jump (Intermediate)
-          # Using a CWJ to get across
+          # Shinespark from Kihunter Room: https://youtu.be/WJ4D_FkPmTA
+          Speed Booster and Shinespark Damage ≥ 60 and Disabled Door Lock Randomizer and Disabled Entrance Randomizer
+          All of the following:
+              Gravity Suit
+              Any of the following:
+                  # Gravity Jump: https://youtu.be/92NyXQFs7K8
+                  Gravity Jump (Intermediate)
+                  # IBJ from floor
+                  Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
+                  # WJ from floor
+                  Hi-Jump Boots and Wall Jumps (Beginner)
+          # CWJ: https://youtu.be/eKBck_F8IVk
           Continuous Wall Jump (Expert) and Movement (Advanced)
           # HBJ across
           Horizontal Infinite Bomb Jump (Advanced) and Can Lay Bombs
   > Pickup (Missile Expansion)
       Any of the following:
           Grapple Beam or Space Jump or Movement (Beginner)
-          # WJ from Floor
-          Gravity Suit and Wall Jumps (Beginner)
-          # Short charge shinespark
-          Speed Booster and Short Charge (Intermediate) and Disabled Door Lock Randomizer and Disabled Entrance Randomizer
+          All of the following:
+              # From Floor
+              Gravity Suit
+              Any of the following:
+                  # Gravity or Wall Jumps
+                  Gravity Jump (Beginner) or Wall Jumps (Beginner)
+                  # IBJ
+                  Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
+          # Shinespark from Kihunter Room: https://youtu.be/WJ4D_FkPmTA
+          Speed Booster and Shinespark (Intermediate) and Shinespark Damage ≥ 50 and Disabled Door Lock Randomizer and Disabled Entrance Randomizer
           # HIBJ
-          Horizontal Infinite Bomb Jump (Beginner) and Can Lay Bombs
+          Horizontal Infinite Bomb Jump (Advanced) and Can Lay Bombs
```

Comment:
i dont think that HBIing to the pickup should be the same difficulty as HBIing all the way across.

Reply:
Ideally you HBJ to the item, and then start a new HBJ across. I'll bump the trick up because it does requires doing a tough trick twice in a row.